include/constants/vars.h

```diff
@@ -94,7 +94,7 @@
 #define VAR_POKELOT_PRIZE_PLACE              0x404D
 #define VAR_UNUSED_0x404E                    0x404E // Unused Var
 #define VAR_LOTAD_SIZE_RECORD                0x404F
-#define VAR_0x4050                           0x4050
+#define VAR_LITTLEROOT_STATE                 0x4050
 #define VAR_ROUTE102_ACCESSIBLE              0x4051
 #define VAR_UNUSED_0x4052                    0x4052 // Unused Var
```

Comment:
`VAR_DEWFORD_TOWN_STATE`

Reply:
That's inconsistent with the other var names. Should I rename the rest as well?

Reply:
Sure. Also, find a different name for `VAR_SOOTOPOLIS_STATE` because the var currently named `VAR_RAYQUAZA_STATE` is logically the Sootopolis State variable.